packages/rest/src/lib/RequestManager.ts

```diff
@@ -389,8 +402,6 @@ export class RequestManager extends EventEmitter {
 
 			// Set the final body to the form data
 			finalBody = formData;
-			// Set the additional headers to the form data ones
-			additionalHeaders = formData.getHeaders();
```

Comment:
Is this not needed anymore? 👀 Does form-data still work?

Reply:
Undici exports its own `FormData`, is there a requirement on form-data?

Reply:
No no as in do the headers get automatically added for content type when form data is sent? Aka does this change work with the api

Reply:
Yes, undici does all of this for us 😄